README.md

```diff
@@ -1,11 +1,8 @@
-
 # React If
 
 [![npm badge](http://img.shields.io/npm/v/react-if.svg)](https://www.npmjs.com/package/react-if)
-[![Build Status](https://travis-ci.org/romac/react-if.svg?branch=master&style=flat)](https://travis-ci.org/romac/react-if)
```

Comment:
please reimplement for github actions kthx

Reply:
Covered in 2109d182ed23c84c46d8d3a0c39ce24d6b81857d